src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -421,7 +413,7 @@ uct_rc_mlx5_iface_init_rx(uct_rc_iface_t *rc_iface,
 
     status = uct_rc_iface_init_rx(rc_iface, rc_config, &iface->rx.srq.verbs.srq);
     if (status != UCS_OK) {
-        return status;
+        goto err;
```

Comment:
is "uct_rc_mlx5_destroy_srq" the opposite of "uct_rc_iface_init_rx"?
since it's called if uct_ib_mlx5_srq_init() fails..

Reply:
yes